code/datums/elements/strippable.dm

```diff
@@ -145,6 +152,13 @@
 	user.log_message("[key_name(source)] is being stripped of [item] by [key_name(src)]", LOG_ATTACK, color="red", log_globally=FALSE)
 	item.add_fingerprint(src)
 
+	if(ishuman(source))
+		var/mob/living/carbon/human/victim_human = source
+		if(victim_human.key && !victim_human.client) // AKA braindead
+			if(victim_human.stat <= SOFT_CRIT && LAZYLEN(victim_human.afk_thefts) <= AFK_THEFT_MAX_MESSAGES)
+				var/list/new_entry = list(list("[user] tried unequipping your [item]", world.time))
```

Comment:
```suggestion
				var/list/new_entry = list(list("[user] tried unequipping your [item.name]", world.time))
```

That way you get "unequipping your disabler" instead of "unequipping your the disabler".